modules/reference/pages/systems/system-details/sd-overview.adoc

```diff
@@ -0,0 +1,179 @@
+[[s5-sm-system-details-overview]]
+= Overview
+
+The menu:Systems[Details > Overview] section shows a summary of information about the selected client, including the current system status.
+This subtab is displayed by default when you click the name of a client anywhere throughout the {webui}.
+
+Click btn:[Delete System] to delete the system profile.
+Deleting a system profile will not destroy or remove the client, but you will no longer be able to manage it with {productname}.
+If you delete a system profile by mistake, you can re-register the client using bootstrap.
+
+The menu:Systems[Details > Overview] subtab is split into sections:
+
+
+
+== System Status
+
+The section shows the current status of the client.
+If no action is required for this client, the status is ``System is up to date``.
+
+If the client has new software ready for installation, the status is ``Software Updates Available``, along with details of updates.
+Apply updates by navigating to menu:System Details[Packages].
+
+
+
+== System Info
+
+This section shows general information about the client.
+
+[[sdoverview-sysinfo-columns]]
+.System Info Columns
+[cols="1,1,1", options="header"]
+|===
+
+| Column Name
+| Description
+| Conditions
+
+| Hostname
+| The hostname used by the client with the {productname} Server.
+| -
+
+| FQDN
+| The fully qualified domain name of the client.
+| Shown only if it differs from the host name.
+//?? --LKB 2020-07-14
+
+| IP address
+| The IPv4 IP address of the client.
+| -
+
+| IPv6 address
+| The IPv6 IP address of the client.
+| -
+
+| Minion ID
+| The minion ID of the client.
+| Salt clients only.
+
+| Virtualization
+| The type of virtualization used by the client.
+| Virtualized clients only.
+
+| UUID
+| The universally unique identifier.
+| -
+
+| Kernel
+| The kernel currently in use on the client.
+| -
+
+| {productname} System ID
+| A unique identifier generated by {productname} when the client is registered.
+| -
+
+| Activation key
+| The activation key used to register the client.
+| Remains blank if the client has not been activated.
+
+| Installed products
+| Software products currently installed on the client.
+| -
+
+| Lock status
+| Indicates whether the client is locked.
+| Traditional clients only.
+Locked Salt clients instead show a yellow banner on this page.
+
+|===
+
+For more information about system locks, see xref:client-configuration:system-locking.adoc[].
+
+
+
+== Subscribed Channels
+
+This section shows the base and child software channels this client is subscribed to.
+
+Click btn:``Alter Channel Subscriptions`` to change the channel subscriptions for this client.
+Click a channel name to go to the [guimenu]``Basic Channel Details`` section.
```

Comment:
```suggestion
Click a channel name to go to the [guimenu]``Channel Details`` tab.
```
``Basic Channel Details`` is one section of the ``Channel Details`` page or tab.